index.html

```diff
@@ -15,10 +15,10 @@ <h1 style="font-size: 20px">Train  layout designer / maker by François CREVOLA
     For more informations and instructions of use, see the <a href="tutorial.html">online help and watch the video tutorial</a> or <a href="http://www.flickr.com/photos/46225254@N00/sets/72157630210133604/">see example of what can done</a>. (Contact me at francois (AT) crevola (DOT) org)
     </div>
     <div>
-        <canvas id="geocommand" width="100" height="920" ><img src="geotrax_layout_no_canvas.png" ></canvas>
-        <canvas style="position: absolute; z-index: -1; " id="geocanvasbg" width="1800" height="920" ><img src="geotrax_layout_no_canvas.png" ></canvas>
-        <canvas style="position: absolute; z-index: 0;" id="geocanvas" width="1800" height="920" ><img src="geotrax_layout_no_canvas.png" ></canvas>
-        <canvas style="position: absolute; z-index: 1; " id="geocanvasfg" width="1800" height="920" ><img src="geotrax_layout_no_canvas.png" ></canvas>
+        <canvas id="geotracklist" width="100" height="920" ><img src="images/geotrax_layout_no_canvas.png" ></canvas>
+        <canvas style="position: absolute; z-index: -1; " id="geolayouteditorbg" width="1800" height="920" ><img src="images/geotrax_layout_no_canvas.png" ></canvas>
```

Comment:
- moving to "images/" is a refactoring not needed for you feature
- changing name of existing canvas is not necessary